### PR TITLE
Improve Home page

### PR DESCRIPTION
**Pull request by e-aakash**
Backend changes for https://github.com/pulsejet/cerium/pull/4

Sends all forms (ids, name & response token) when requested in /api/forms end point

### CODE REVIEW

File: controllers/form.go [hunk 1]
```diff
@@ -127,3 +129,59 @@ var GetForm = func(w http.ResponseWriter, r *http.Request) {
...
+	count, err := collection.CountDocuments(u.Context(), bson.M{"creator": rno})
+	if count == 0 {
+		u.Respond(w, forms, 200)
+	}
+
+	// To Set which fields are required in the output
+	type fields struct {
+		ID		int		`bson:"_id"`
+		Name 	int		`bson:"name"`
+		Token 	int		`bson:"responsetoken"`
```
Comment (pulsejet) [inline controllers/form.go]: This indentation is strange. Use the latest VS Code with the official golang extension and install the required modules to ensure this doesn't happen.

**Comment by pulsejet** [comment]
You might want to query the DB to give you forms in order of descending created date, since this makes most sense practically.

File: controllers/form.go (hunk 1 above)
Comment (e-aakash) [reply controllers/form.go]: Fixed, thanks for pointing out.

**Review by pulsejet — APPROVED** [review]
Awesome!

**Pull request merged**